src/ClientSideMonitoring/AbstractMonitoringMiddleware.php

```diff
@@ -224,6 +224,23 @@ protected function populateResultEventData(
         return $event;
     }
 
+    
+    /**
+     * Checks if the socket is created.
+     *
+     * @return bool Returns true if the socket is created, false otherwise.
+     */
+    private function isSocketCreated()
+    {
+        // Before version 8, sockets are resources
+        // After version 8, sockets are instances of Socket
+        if (PHP_MAJOR_VERSION >= 8) {
+            return !(self::$socket instanceof \Socket);
```

Comment:
Thanks for the previous change, however, there is something else we would need to address and is that I think the expression should be:
```php
return (self::$socket instanceof \Socket);
```
instead of:
```php
return !(self::$socket instanceof \Socket);
```
because the function will return false even when the socket is already created, since you are prefixing the expression with a negation symbol.

What do you think?

Reply:
You're right, the logic was inverted - have fixed it now